src/client/java/pom.xml

```diff
@@ -36,42 +36,38 @@
     <daos.install.path>/usr/local/daos</daos.install.path>
   </properties>
 
-  <repositories>
-    <repository>
-      <id>maven2-repo.dev.java.net</id>
-      <name>java.net repository for Maven</name>
-      <url>http://download.java.net/maven/2</url>
-      <layout>default</layout>
-    </repository>
-  </repositories>
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Jiafu Zhang</name>
```

Comment:
Is it a must to add developer here? If not, can we remove it? Otherwise what about only keep the organizationUrl like below as this is an open source project? 
```
<organizationUrl>http://daos.io</organizationUrl>
```

Reply:
I think I can remove it. It's added by following the guide.